source/common/http/filter/fault_filter.cc

```diff
@@ -21,47 +21,27 @@ FaultFilterConfig::FaultFilterConfig(const Json::Object& json_config, Runtime::L
 
   json_config.validateSchema(Json::Schema::FAULT_HTTP_FILTER_SCHEMA);
 
-  const Json::ObjectPtr abort = json_config.getObject("abort", true);
-  const Json::ObjectPtr delay = json_config.getObject("delay", true);
+  const Json::ObjectPtr config_abort = json_config.getObject("abort", true);
+  const Json::ObjectPtr config_delay = json_config.getObject("delay", true);
 
-  if (abort->empty() && delay->empty()) {
+  if (config_abort->empty() && config_delay->empty()) {
```

Comment:
potentially same thing with the route rules, where it's either prefix or path needs to be supplied:
```
     bool has_prefix = route->hasObject("prefix");
     bool has_path = route->hasObject("path");
     if (!(has_prefix ^ has_path)) {
       throw EnvoyException("routes must specify either prefix or path");
     }
```